src/base/static/components/place-detail/index.js

```diff
@@ -70,6 +71,23 @@ class PlaceDetail extends Component {
     };
   }
 
+  onMountTargetResponse(responseRef) {
+    setTimeout(() => {
```

Comment:
Why is this timeout needed?? I'm not sure. Without it, though, all values in the result from `getBoundingClientRect()` are 0, which results in no scrolling.

Reply:
It turns out that this will work as expected with a 0-length timeout. I'm not sure exactly as to why, but it sounds like the reason might have to do with allowing the browser to complete other queued rendering tasks before attempting to access the DOM node and its `boundingClientRect`. 

This post has a lot of information:

https://stackoverflow.com/questions/779379/why-is-settimeoutfn-0-sometimes-useful/23747597#23747597

From what I can tell the 0-length timeout trick is hacky but accepted. Maybe we should use that trick here?

Reply:
Very interesting - thanks for looking into this! I think this hack is reasonable, but let's add a comment just to explain why.

Also, have you tried solving this with `requestAnimationFrame`? https://developer.mozilla.org/en-US/docs/Web/API/window/requestAnimationFrame I'm not sure if one provides a better guarantee over the other, but it seems like `requestAnimationFrame` exists for these types of situations.

Reply:
Yes, confirm that `requestAnimationFrame()` does the same job as a 0-length timeout. I'll go ahead and use that implementation (along with a comment explaining the need for it).